lib/targets/hip/comm_target.cpp

```diff
@@ -31,10 +31,10 @@ namespace quda
     return std::max(accessRank[0], accessRank[1]);
   }
 
-  void comm_create_neighbor_memory(void *remote[QUDA_MAX_DIM][2], void *local)
+  void comm_create_neighbor_memory(array_2d<void8, QUDA_MAX_DIM, 2> &remote, void *local)
```

Comment:
Oopsie void8 is not a type. I am fixing in my copy... just in case there are other issues.....